GPU/D3D11/TextureCacheD3D11.cpp

```diff
@@ -399,10 +399,11 @@ void TextureCacheD3D11::ApplyTextureFramebuffer(TexCacheEntry *entry, VirtualFra
 		shaderApply.ApplyBounds(gstate_c.vertBounds, gstate_c.curTextureXOffset, gstate_c.curTextureYOffset, xoff, yoff);
 		shaderApply.Use(depalShaderCache_->GetDepalettizeVertexShader(), depalShaderCache_->GetInputLayout());
 
+		ID3D11ShaderResourceView *nullTexture = nullptr;
+		framebufferManagerD3D11_->BindFramebufferAsColorTexture(0, framebuffer, BINDFBCOLOR_SKIP_COPY | BINDFBCOLOR_FORCE_SELF);
```

Comment:
Hm, I thought this could fail if the current target was framebuffer?

-[Unknown]

Reply:
Argh, yeah, I guess we just get the opposite problem. I'll set to null first.